radar/qa/ardoq.yaml

```diff
@@ -0,0 +1,18 @@
+name: Ardoq
+blip:
+  - date: 2019-06-20
+    ring: ASSESS
+description: |
+  Ardoq is a collaborative, easy-to-use and intuitive tool for documenting your projects,
+  business processes, applications, services, infrastructure or anything else that you can document in a structured way.
+rationale: |
+  We have identified a gap within our technical documentation.
+  Detail of dependencies and architecture if low and process diagrams are either lacking or not updated.
+  We need a tool that assists capturing this information, with minimum effort and maximum value to an audience.
+  As RS is about to go into a huge phase of change on an architecture level
```

Comment:
Not only RS, I would change it to the organisation.

Reply:
should be fixed now.